spec/integrity/i18n_spec.rb

```diff
@@ -37,8 +37,10 @@ def load_yaml(path)
   end
 
   it "has an i18n key for each Badge description" do
+    # skip non core badges
     Badge
       .where(system: true)
+      .where("id < 100")
```

Comment:
I'd rather fix plugins. Do you know which plugin was causing problems?

Reply:
discourse-solved I think is one of them